data/json/items/gun/22.json

```diff
@@ -439,7 +439,7 @@
     "variants": [
       {
         "id": "ruger_charger",
-        "name": { "str": "Ruger 22 Charger" },
+        "name": { "str": "Ruger 22 machine pistol" },
```

Comment:
Machine pistol is a specific term for automatic pistols. The Charger is semi-automatic